recipes/fuzzymatcher/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "fuzzymatcher" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+
+  sha256: 24fd3bb3c347523802af378e711aef48200759bc498546599a0652f39f3e9242
+
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - pandas >=0.20.3
+    - metaphone
+    - fuzzywuzzy
+    - python-dateutil
+
+test:
+  imports:
+    - fuzzymatcher
```

can we try to add ``pip check`` here please?
```suggestion
    - fuzzymatcher
  requires:
    - pip
  commands:
    - pip check
```